contracts/LiquidPledgingBase.sol

```diff
@@ -1,89 +1,128 @@
 pragma solidity ^0.4.11;
+/*
+    Copyright 2017, Jordi Baylina
+    Contributor: Adrià Massanet <adria@codecontext.io>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
 
 import "./ILiquidPledgingPlugin.sol";
 import "../node_modules/giveth-common-contracts/contracts/Owned.sol";
 
-/// @dev This is declares a few functions from `Vault` so that the
-///  `LiquidPledgingBase` contract can interface with the `Vault` contract
-contract Vault {
+/// @dev `LPVault` serves as an interface to allow the `LiquidPledgingBase`
+///  contract to interface with a `LPVault` contract
+contract LPVault {
     function authorizePayment(bytes32 _ref, address _dest, uint _amount);
     function () payable;
 }
 
+/// @dev `LiquidPledgingBase` is the base level contract used to carry out
+///  liquid pledging's most basic functions, mostly handling and searching  the
+///  data structures
 contract LiquidPledgingBase is Owned {
+
     // Limits inserted to prevent large loops that could prevent canceling
     uint constant MAX_DELEGATES = 20;
     uint constant MAX_SUBPROJECT_LEVEL = 20;
     uint constant MAX_INTERPROJECT_LEVEL = 20;
 
     enum PledgeAdminType { Giver, Delegate, Project }
-    enum PaymentState { Pledged, Paying, Paid } // TODO name change Pledged
+    enum PaymentState { Pledged, Paying, Paid }
 
-    /// @dev This struct defines the details of each the PledgeAdmin, these
-    ///  PledgeAdmins can own pledges and act as delegates
-    struct PledgeAdmin { // TODO name change PledgeAdmin
+    /// @dev This struct defines the details of the `PledgeAdmin` which are 
+    ///  Referenced by their id and can own pledges and act as delegates
```

Comment:
what do you think about changing "by their id" to something like "by their index in the admins array". Or adding something regarding how their id is assigned b/c it's not in the struct which this is commenting

Reply:
Yeah I changed it, it still seems worded a little awkward but I think it's important to note. Basically the only place the struct is used is in the `admins` object or to add things to the `admins` object.